sys/include/net/ipv6/addr.h

```diff
@@ -816,10 +816,20 @@ static inline char *ipv6_addr_split_iface(char *addr_str)
 /**
  * @brief Print IPv6 address to stdout
  *
- * @param[in]   addr  Pointer to ipv6_addr_t to print
+ * @param[in]   addr    Pointer to ipv6_addr_t to print
  */
 void ipv6_addr_print(const ipv6_addr_t *addr);
 
+/**
+ * @brief Print IPv6 addresses to stdout
+ *
+ * @param[in]   addrs       Array of addresses to print
```

Comment:
```suggestion
 * @param[in]   addr       Array of addresses to print
```

Reply:
Nice catch! I updated the function name instead (since this function actually works with possibly multiple addresses, it might be more intuitive to read a plural there).